src/styles/calendar.scss

```diff
@@ -0,0 +1,53 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-size;
+  width: $calendar-size;
+  padding: 10px;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    box-sizing: border-box;
+    height: 100px;
+    width: 100px;
+
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    background-color: #eee;
+    border: 1px solid black;
+
+    &:hover {
+      transition-property: all 0.5s;
```

Comment:
The 'transition-property' should be specified with the properties that the transition effect should work with instead of using 'all'. This can enhance performance as the browser knows exactly which properties to watch for changes.